airbyte-config/config-models/build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
 dependencies {
     implementation project(':airbyte-json-validation')
     implementation project(':airbyte-protocol:protocol-models')
+    implementation project(':airbyte-api')
```

Comment:
From parent PR: ![image](https://user-images.githubusercontent.com/7798112/186285365-84fc53ff-15ce-4197-b9f6-310d164820c4.png)

Reply:
This is going away, moving discussion to larger thread here: https://github.com/airbytehq/airbyte/pull/15902/files#r953189950